src/pages/observability-packs.js

```diff
@@ -108,7 +138,27 @@ const ObservabilityPacksPage = ({ data, location }) => {
     setQuerySort,
     setQuerySearch,
   ]);
-
+  useEffect(() => {
+    const handleClickThrough = () => {
+      tessen.track('observabilityPack', `ObservabilityPackClickThroughView`, {
+        oPackClickThroughView: view,
+      });
+      if (typeof window !== 'undefined' && window.newrelic) {
+        window.newrelic.addPageAction('oPackClickThroughView', {
+          oPackClickThroughView: view,
+        });
+      }
+    };
+    document.querySelectorAll('.pack-tile-instrument').forEach((item) => {
```

Comment:
I could be wrong, but can we just add an `onClick` handler to the `Surface` component itself?

Reply:
i did try that in a few different ways but it just seems to do nothing, and I couldn't figure out why 🤷 will campfire this monday to see if there are other options